### PR TITLE
[#490] Doc updated in context of snap store

### PR DESCRIPTION
**Pull request by AakashMallik**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug fix
- [ ] Feature implementation
- [x] Doc updates

* **What changes have you introduced?**
- Doc updated in the context of snap store

* **Does this PR introduce a breaking change?**
No

### CODE REVIEW

**Comment by mariobehling** [comment]
Please also add a png for the PSLab Banner in /docs/images. Keep the SVG as a reference in the docs/images folder, please. Github does not display svgs. They deactivated it. Please use the /docs/images folder for images.

**Comment by AakashMallik** [comment]
@mariobehling Addressed the image issue

File: README.md [hunk 1]
```diff
@@ -2,16 +2,15 @@
...
 [![Gitter](https://badges.gitter.im/fossasia/pslab.svg)](https://gitter.im/fossasia/pslab?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/8259e5c2220f484e95a88cf4aaed1a96)](https://www.codacy.com/app/mb/pslab-desktop?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=fossasia/pslab-desktop&amp;utm_campaign=Badge_Grade)
 [![Mailing List](https://img.shields.io/badge/Mailing%20List-FOSSASIA-blue.svg)](https://groups.google.com/forum/#!forum/pslab-fossasia)
 [![Twitter Follow](https://img.shields.io/twitter/follow/pslabio.svg?style=social&label=Follow&maxAge=2592000?style=flat-square)](https://twitter.com/pslabio)
-[![pslab](https://snapcraft.io/pslab/badge.svg)](https://snapcraft.io/pslab)
 
-<a href="https://pslab.io/" rel="some text">![PSLab Banner](./docs/project_banner.png)</a>  
+<a href="https://pslab.io/" rel="PSLab Link">
+	<img src="./docs/images/project_banner.jpg" alt="PSLab Banner" width="700px" height="200px"/>
+</a>
```
Comment (CloudyPadmal) [inline README.md]: Add a line break here

Comment (AakashMallik) [reply README.md]: Yeah.. just saw it... have updated it... I'll rebase that commit in a while

Comment (AakashMallik) [reply README.md]: Or you can squash and merge right away... should be fine

File: README.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ Electron desktop application for an awesome open-hardware platform. Made with lo
 <a href="https://pslab.io/" rel="PSLab Link">
 	<img src="./docs/images/project_banner.jpg" alt="PSLab Banner" width="700px" height="200px"/>
 </a>  
+
+***
```
Comment (CloudyPadmal) [inline README.md]: Let's remove this hr

Comment (AakashMallik) [reply README.md]: Done sir...

Comment (AakashMallik) [reply README.md]: @CloudyPadmal  here is the preview https://github.com/AakashMallik/pslab-desktop/tree/doc_update_snap

**Pull request merged**